pkg/features/feature_manager.go

```diff
@@ -0,0 +1,265 @@
+/*
+ * Copyright 2022 Singularity Data
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package features
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"unicode"
+)
+
+type FeatureStage string
+type FeatureName string
+
+const (
+	EnableOpenKruiseFeature FeatureName = "enableOpenKruise"
+)
```

Comment:
Looks like it duplicates with the `SupportedFeatureList`. I think we should remove it. It's fine to just use `string`.